src/components/Moviecontainer/Moviecontainer.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react'
+import Movieitem from '../Movieitem/Movieitem'
+import './Moviecontainer.scss'
+import { Movie } from '../../common/types/movie'
+
+interface Props {
+    movieList: Movie[]
+}
+
+const Moviecontainer: React.FC<Props> = (props) => {
```

Comment:
Kind of a new trend nowadays, is to ditch the usage of `React.FC`. It might save you whole lotta pain further on. The new advice on this topic is:

`export const MovieContainer = (props: MovieContainerProps) => { ... }`

Let the reurn type be inferred by TS. Also, when you mess up something, and try to render it in another component's context, it will complain anyways :)

Another idea here, basing on the before mentioned - use destructuring when possible:
`export const MovieContainer = ({ movieList }: MovieContainerProps) => { ... }`